README.md

```diff
@@ -161,4 +161,4 @@ CloudGraph AWS Provider will ask you what regions you would like to crawl and wi
 | vpc                         | alb, codebuild, dmsReplicationInstance, ec2, eip, elb, ecsService, efsMountTarget, eksCluster igw, elastiCacheCluster, elasticSearchDomain, lambda, nacl, natGateway, networkInterface, rdsClusterSnapshot, rdsDbInstance, redshiftCluster, route53HostedZone, routeTable, subnet, flowLog, vpnGateway, transitGatewayAttachment                                              |
 | vpnConnection               | customerGateway, transitGateway, transitGatewayAttachment, vpnGateway                                                                                                                                                                                                                                                                                                         |
 | vpnGateway                  | vpc, vpnConnection                                                                                                                                                                                                                                                                                                                                                            |
-| wafV2WebAcl                 | appSync                                                                                                                                                                                                                                                                                                                                                                       |
+| wafV2WebAcl                 | appSync, apiGatewayStage, alb                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Are we able to add an elb connection too or does it only connect to alb?

Reply:
According to the docs, it only connects to application load balancers (alb)